django/src/rdwatch/views/model_run.py

```diff
@@ -495,7 +495,7 @@ def check_download(request: HttpRequest, task_id: str):
 
 
 @router.get('/{id}/download')
-def get_downloaded_annotations(request: HttpRequest, id: int, task_id: str):
+def get_downloaded_annotations(request: HttpRequest, id: str, task_id: str):
```

Comment:
```suggestion
def get_downloaded_annotations(request: HttpRequest, id: UUID4, task_id: str):
```

Reply:
I had that initially but noticed that the `AnnotationExport` model had a charfield even though it is a UUID.  Should we update that model to a uuid instead of a str?

Reply:
Oh, I didn't realize. That `id` isn't even used, it's only the `task_id` is (which is indeed a `str` and not `UUID4`). Should we just remove it?